en/includes/guides/authorization/user-impersonation/user-impersonation-via-business-app.md

```diff
@@ -1,28 +1,107 @@
+    Starting impersonation through a business application requires some manual, advanced steps. The recommended method is to impersonate users via the Console. However, your organization must meet the following requirements to do this:
+
+    - Your organization has [enabled]({{base_path}}/guides/user-self-service/configure-self-service-portal/#enabledisable-the-my-account-portal) the My Account portal.
+
+    - Your {{product_name}} Console and My Account portal have the same login flow.
+
+    If your organization can meet these requirements, follow the guide to [start impersonation from the Console]({{base_path}}/guides/authorization/user-impersonation/via-console/).
+
+## Step 1: Configure business applications for impersonation
+
+To let an impersonator access a business application as another user, you need to enable impersonation for that app. This section shows you how to set it up.
+
+### Step 1.1: Authorize application to use the impersonation API
+
+By following the steps below, you permit a business application to use the impersonation API.
+
+1. On the {{ product_name }} Console, go to **Applications**.
+
+2. Select your application and go to its **API Authorization** tab.
+
+3. Click **Authorize an API Resource** and do the following:
+
+    1. Under **API Resource**, select **User Impersonation**.
+
+    2. Under **Authorized Scopes**, select **User Impersonation Scope**.
+
+    3. Click **Finish**.
+
+    ![Authorize impersonation API]({{base_path}}/assets/img/guides/authorization/impersonation/api-authorization-impersonation.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+### Step 1.2: Create an application role that permits impersonation
+
+To access business applications as an impersonated user, you need to create a new role to give the impersonator the right permissions.
+
+To do so,
+
+1. On the {{product_name}} Console, go to **Applications** and select your application.
+
+2. Go to the **Roles** tab of the application.
+
+3. Under **Role Audience**, select **Application**. This sets the audience of this role to the current application.
+
+3. Click **New Role** and do the following:
+
+    1. Provide a suitable role name.
+
+    2. Under **API Resource**, select **User Impersonation**.
+
+    3. Select the checkbox corresponding to **User Impersonation** to select its scopes.
+
+    4. Click **Create**.
+
+        ![Role-Creation]({{base_path}}/assets/img/guides/authorization/impersonation/role-creation.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+### Step 1.3: Assign the role to the impersonator
+
+To assign the role,
+
+1. On the {{product_name}} Console, go to **User Management** > **Roles**.
+
+2. Select the role you just created. The audience of this role is set to your business application.
+
+3. Under the **Users** tab, click **Assign Users** and select the impersonator.
+
+4. Click **Update** to save the changes.
+
+
```

Comment:
Remove the additional line.

Reply:
Fixed with e00b6f57483724a5af53e136aae9546d26348f52